CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ This release extends the planned support of the modules to OneView REST API vers
 #### Major changes
 1. Extended support of planned modules to API800.
 2. Updated library files to support Python SDK changes.
-3. Modules upgraded in this release required hpOneView version 5.0.0 or above.
+3. Modules upgraded in this release requires hpOneView version 5.0.0b0 or above.
```

Comment:
Hi @sijeesh - is this adhere to [semantic versioning 2.0](https://semver.org/)

Reply:
This version number is generated by Pypi.
The actual version number of the SDK is 5.0.0-beta.
https://github.com/HewlettPackard/python-hpOneView/blob/release/5.0.0-beta/CHANGELOG.md

Reply:
got it..thanks for the clarification.